.github/ISSUE_TEMPLATE/bug-core.md

```diff
@@ -1,13 +1,11 @@
 ---
+
```

Comment:
There is a lot of diff in this PR that you probably did not change?  Rebase off Main to get rid of the diff if you can.  Thanks.